work/src/org/kuali/kfs/module/ar/businessobject/lookup/TicklersReportLookupableHelperServiceImpl.java

```diff
@@ -134,7 +130,21 @@ public Collection performLookup(LookupForm lookupForm, Collection resultTable, b
             // Check for followup date range
             boolean isValid = true;
 
-            if (!StringUtils.isBlank(agencyNumber)) {
+            if (StringUtils.isNotBlank(completed)) {
+                if(StringUtils.equalsIgnoreCase(completed, "Y")) {
```

Comment:
Kevin, could you Use KRADConstants.YES_INDICATOR_VALUE and NO_INDICATOR_VALUE here?